smarts/core/default_map_builder.py

```diff
@@ -128,6 +142,9 @@ def get_road_map(map_spec) -> Tuple[Optional[RoadMap], Optional[str]]:
         _clear_cache()
 
     road_map = map_class.from_spec(map_spec)
+    if road_map is None:
+        return None, None
```

Comment:
`WaymoMap` can return None if the `map_spec.source` isn't valid. So to avoid CI to fail, this check was needed otherwise we return valid hash for invalid road_map.

Reply:
Yes, there was a case where this could return `None`, which was causing `build-all` to throw an error. See `_is_scenario_folder_to_build()` in studio.py for more context.